.github/replay-soc2.yml

```diff
@@ -0,0 +1,15 @@
+---
+# This is the block list used for the Replay SOC2 bot.
+#
+# An example:
+# blocklist:
+#  - bad/**
+#  - rootfile
+#
+# Each line maps to a glob expression that gets applied to the list of files
+# that the PR touches. The Replay SOC2 bot will never approve any changes to
+# .github/replay_soc2.yml. The matching is done by minimatch.
+#
+allowlist:
+  - replay-test/**
```

Comment:
Should we add `.github` to this list?

Reply:
The approveman version of this: https://github.com/replayio/node/blob/master/.github/approveman.yml, didn't have .github . But I can add it. :)